winterfell/webapp/src/js/profileService.js

```diff
@@ -1,12 +1,47 @@
 'use strict';
 var app = angular.module('vetafiApp');
-app.factory('profileService', ['claimService', function(claimService) {
-  return {
-    userInfo: {},
-    clearUserInfo: function() {
-      sessionStorageHelper.removePair(vfiConstants.keyUserId);
-      this.userInfo = {};
+app.service('Profile', ['claimService', '$q', 'net', function(claimService, $q, net) {
```

Comment:
oh my bad. I guess I could've sworn I saw something depend on both profileService and Profile. Nevermind then.